pkg/build/controller/strategy/util.go

```diff
@@ -160,6 +162,46 @@ func mountSecretVolume(pod *v1.Pod, container *v1.Container, secretName, mountPa
 	container.VolumeMounts = append(container.VolumeMounts, volumeMount)
 }
 
+// mountConfigMapVolume is a helper method responsible for actual mounting configMap
+// volumes into a pod.
+func mountConfigMapVolume(pod *v1.Pod, container *v1.Container, configMapName, mountPath, volumeSuffix string) {
+	volumeName := apihelpers.GetName(configMapName, volumeSuffix, kvalidation.DNS1123LabelMaxLength)
```

Comment:
Copy-pasted - opportunity to refactor here.

Reply:
yup same as before